PlexPy.py

```diff
@@ -21,20 +16,20 @@
 #  You should have received a copy of the GNU General Public License
 #  along with PlexPy.  If not, see <http://www.gnu.org/licenses/>.
 
-import os
-import sys
-
-# Ensure lib added to path, before any other imports
-sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'lib/'))
```

Comment:
This is the only breaking change and I removed it for development.  Instead of looking in the provided `lib/` dir this looks at the installed libraries.  When this Pull request is up to snuff I imagined updating the `lib/` from the installed library and not dirtying an already nit-picky commit.